### ISSUE TITLE
Error while importing Curl: TypeError: thisHeader.indexOf is not a function

### ISSUE THREAD

**Issue by jkshobby** [a85/POSTMan-Chrome-Extension#5286]
## App Details:

Version 6.3.0 (6.3.0)
```
Postman for Mac
Version 4.11.0
OS X 16.4.0 / x64
```

## Issue Report:
found this import error while importing the cURL commands and trying to run it to postman.

**Comment by ingabraham** [comment]
have the same fucking issue

**Comment by kthulsidoss** [comment]
@ingabraham please kindly refrain using the f word .appreciate very much tx k.thulsi doss

**Comment by abhijitkane** [comment]
@jkshobby @ingabraham Can you provide a sample curl command that reliably reproduces this? We're working on updating the importers, and working on all existing bug reports/feature requests.

**Comment by ingabraham** [comment]
@kthulsidoss sorry for the bad word
@abhijitkane here it is
curl 'https://www.ihg.com/guestapi/v1/web/changeLanguage?brandCode=6c&countryCode=us&langCode=en&targetCountry=CN&targetLanguage=ZH&currentUrl=https%3A%2F%2Fwww.ihg.com%2Fhotels%2Fus%2Fen%2Ffind-hotels%2Fhotel%2Flist%3FqDest%3DAtlanta%2C%2520GA%2C%2520United%2520States%26qCiMy%3D92018%26qCiD%3D2%26qCoMy%3D92018%26qCoD%3D3%26qAdlt%3D1%26qChld%3D0%26qRms%3D1%26qRtP%3DIVEDI.6CBARC%26qCpid%3D954408789%26qIta%3D99504425%26qAkamaiCC%3DUS%26qSrt%3DsDD%26qBrs%3Dic.in.vn.cp.hi.ex.rs.cv.sb.cw.ma.ul.ki.va%26qAAR%3DIVEDI.6CBARC%26srb_u%3D1%26qRad%3D30' -X POST -H 'authority: www.ihg.com' -H 'content-length: 0' -H 'cache-control: max-age=0' -H 'origin: https://www.ihg.com' -H 'upgrade-insecure-requests: 1' -H 'content-type: application/x-www-form-urlencoded' -H 'user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36' -H 'accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8' -H 'referer: https://www.ihg.com/hotels/us/en/find-hotels/hotel/list?qDest=Atlanta,%20GA,%20United%20States&qCiMy=92018&qCiD=2&qCoMy=92018&qCoD=3&qAdlt=1&qChld=0&qRms=1&qRtP=IVEDI.6CBARC&qCpid=954408789&qIta=99504425&qAkamaiCC=US&qSrt=sDD&qBrs=ic.in.vn.cp.hi.ex.rs.cv.sb.cw.ma.ul.ki.va&qAAR=IVEDI.6CBARC&srb_u=1&qRad=30' -H 'accept-encoding: gzip, deflate, br' -H 'accept-language: en-US,en;q=0.9' -H $'cookie: akaas_RBF=2147483647~rv=81~id=c9e0d3e5a0e2ba26ec04735229f0027a; trial=true; aam_tnt=seg6%3DgenericVisitor; aam_uuid=34688296101931443502607912378985488411; cto_lwid=a87a2bf6-a0b6-4803-a825-8835815d2940; WRUIDCD=1786042744341562; _ga=GA1.2.1832031675.1529082544; s_vi=[CS]v1|2D94112B851D5158-600001628000F567[CE]; atgRecVisitorId=1122Ekk1vA51GfrcR5eTxQtgne0mzCceI00emQwaTsIxvXw1A71; aam_interact=seg%3D5929309; _abck=918EF69E7B31F6ADFF1F3B2ADE5E25D117D80637977D0000B5014D5BAD26AD06~0~Dp7OSZVitFUHui0Y05eExDjjcBh+38ZEXui7SL86ZYE=~-1~-1; viewfullsite=true; cto_idcpy=9d2c9c72-db0a-460b-b004-e8845db0e13c; s_fid=4B21CC46D1B9C896-29CF6BD2C8729751; s_dslv=1533736750662; _gcl_aw=GCL.1534165137.EAIaIQobChMIifuavYrq3AIVUYGzCh0ARwLOEAAYASABEgJU9vD_BwE; AAMC_ihg_0=AMSYNCSOP%7C411-17788; recentSearch=cYNX0rvChvz-LkQo9lCBuZn_BsJEWykFiNTug4u5mnAK0BeYIS8H8r6HlAHi5K7k3_RHPbdDzXYyNddmR84uoBrlF8j-kIcCZdkQtW5GUTrpV--ingln0P02HrrP37vbgcJg69ntHcHZEII6yZXpdob5hP0-32aimvInoaQophk2xCfDrrwGK63EUnFlSezK6azbZ6Epv52vXAdkK6ePMtssZObBCikIQxFkmk9viQxTTdXDY_24zGp6VZ6-S3Uhw824zbMkWS6AtRLV60wYq12pZYoyikD1749fCdFCxJ5iZBP88-JQ3dYuOvXUYqWvE43YGHJsPwo3WBAolkpHj8L-Q-TJ3HBd9-GM2t3Oe33WNqPo1D3cUE0HKibEoRyOggxSVyrF-nk8fG2dXxGeykL01mQ_Mhn8qLuocdE63DcHsFf-3yazWJw; _CT_RS_=Recording; UXER=3FVVOAble43gUG5GxISJuPdBxEgswbKFXiVcfM64WAejgqWXaPvG1fg; __atuvc=0%7C34%2C0%7C35%2C0%7C36%2C0%7C37%2C2%7C38; __atssc=google%3B5; _hp2_id.1974984339=%7B%22userId%22%3A%223695927566081100%22%2C%22pageviewId%22%3A%226018767100702960%22%2C%22sessionId%22%3A%223032268536459698%22%2C%22identity%22%3Anull%2C%22trackerVersion%22%3A%224.0%22%7D; experiment_1Rh3wxV09i5nJu430m17=%7B%22experimenting%22%3Afalse%7D; pixlee_analytics_cookie=%7B%22CURRENT_PIXLEE_USER_ID%22%3A%22e76a4ff0-c40c-9add-0a20-a6acae777b7d%22%2C%22CURRENT_PIXLEE_ALBUM_PHOTOS%22%3A%5B%5D%2C%22CURRENT_PIXLEE_ALBUM_PHOTOS_TIMESTAMP%22%3A%5B%5D%2C%22HORIZONTAL_PAGE%22%3A%5B%5D%7D; _tq_id.TV-094572-2.2958=639d8f49f9ac248e.1529081796.0.1537882407..; s_v38=MOSMGC-6C-CN-ZH-GCYR_MFWTG; ctm={\'pgv\':1889908006171628|\'vst\':7051047340200329|\'vstr\':1278332651020161|\'intr\':1537885697382|\'v\':1|\'lvst\':54}; searchIgnite_cookie=99504425; beFree_cookie=99504425; IHGglat=SEAR; beFree_cookie=99504425; QuantumMetricUserID=68875b3c780842a039a11397fda74989; roomKeyCookie=1538138179150; ensUID=27480915RnyYI3HmedOV; _tq_id.TV-094572-1.2958=a31ba9ba10960ca6.1529946369.0.1538403841..; _gcl_au=1.1.1434329718.1538423007; akamaiCountryCode=US; akamaiIsWirelessDevice=false; akamaiIsTablet=false; viewport=large; orientation=landscape; X-IHG-TrueClient_IP=165.2.72.10; BlueStripe.PVN=cd4800001ab5; datacenternode=us-va-iadd1; Origin-Destination=rsp-b; check=true; AMCVS_8EAD67C25245B1870A490D4C%40AdobeOrg=1; s_cc=true; iataCookie=99504425; iataSessionCookie=99504425; iataSessionCookie=99504425; iataCookie=99504425; xdVisitorId=1122Ekk1vA51GfrcR5eTxQtgne0mzCceI00emQwaTsIxvXw1A71; atgPlatoStop=1; fs_nocache_guid=3AFE7D84A17E607D3A319A0F8D8EA8E5; recentlyViewed=CCSHA**DENBO**MRCHB**MRCCP; TVSquared=true; JSESSIONID=1da6oqr7s84z71ft38dz161fzn; ihg_sel_tpid=195404755; JSESSIONID=11177F385C5532336C944AA191FF375C; s_sq=%5B%5BB%5D%5D; ibean=F-IKZtC45C-ksRwoy-IzH3oxb9Il8zQO-Mk6QjHx8gXZzuW55PmaBCsqNUmgNzYIqXIRVFkbgNz2nHeg9LhHU_wcTxPED1cTI-qvDbFs535oLLcIuj1nWMW2Q7ztgU9VfKSClMOBkcoownX-wrphOTVUTwMW5EBAw9QrQNNyxT_rdbuHPSqHUsmvaae6YqgA1VylFD7KEdGlihC5m9qJ7R18O_aXIRuEA7jz6TE5jKhH7JBhE2MM91BWhiuw3SwbVajHhGG3gql6LpLawbKAkn-chuQKcPto8jEzF-I8bOBeLxUFPSeYnq8foi9KfscGK_FNaZ3RvqmOZQtZaSHnCZ4sgZ9RHGQ; s_vi_s=1538493910; ta_upcomres=true; s_nr=1538494878518-Repeat; viewfullsite=true; beFreeCookieCreationDate=Oct-3-2018; fltk=SegID%3D9921698%2CSegID%3D9922058%2CSegID%3D9922184%2CSegID%3D9922229%2CSegID%3D9922232%2CSegID%3D9922235%2CSegID%3D9922237%2CSegID%3D9922238%2CSegID%3D9922243%2CSegID%3D9922245%2CSegID%3D9921966%2CSegID%3D10283208%2CSegID%3D3509550%2CSegID%3D7145167%2CSegID%3D3508962%2CSegID%3D3509454%2CSegID%3D3529171%2CSegID%3D3510809%2CSegID%3D3529172%2CSegID%3D3577290%2CSegID%3D3577321%2CSegID%3D3577242%2CSegID%3D11807395; country_language="us$:es"; beFreeCookieCreationDate=Oct-03-2018; SMIDENTITY=hAC4d7yYNOocEvhilXbmVoY+SWZoYLVtI0BmWa1eFLh1X8swpRXRe7/ri3bMDrdExeQtFrVo0A4FDSWVEnamTNRBEDzebB/gbzKfF5e2Kr32/JSz45YuZFCALYPb+wVMRTSHhZRBMm2nuXzjW0N5jNpQNvRxh+ypPyipHOejyn1mWCm9nRTVlGh6u3RT/YXxN2ffUG05TtwbphbS6/dtpERcDu0onP2hnvnS/LCeOdG2QX2OqdylTXiWf6Xz1E5fNWlsv7nRrNx2JVs2sgAnE9JTYeaU+9vhbWuUoCYMRFCVl28JGItWTNR+Guqrps5F1ZKxLshoELJvhKshb/LbrqLFqRbtAh3sPApZb9wR6uMsSNiHSe7+/pqjiilanhZO0HQCLHflCuPXRkT07puymirL24bgrbQMClx+alGW4Mp/rq4sG3sGrlD/nXTgJoQc1MkvWyPRZhhdxaWgOUt8VCTsXVoqYrwPD0Y8DNW+JhIwSgmZ87EzBMXmzt3Xp+2UggEvnK3a+Aqkk0ETZ6jfd+5loWr4xRirbj86h4pPbjh32PBKNoDSKXlFcY08oMX33fH8nrs3DUrUXX0J7zBiLSpG7XRJTkQ+r+CsoOsSnWSZE42oHm1lzYgXOgXdsYCggjuQvZz2Wduj4SsbYAzWvgWLUZIrq+OyAy6EjKNRcSsA1H19ZM3Oan7FKMGqN+KLNk7WliA2FCYnWlgrrlWCI7d4zZO396GXvMTXMesgOpgofGNjl7it1/MmJmIHdiSuGWRXcNY+kUxtekwcRPJB67Ek07iv7iigoRuIvMzM69D2GODSR0pw4tZPwbxtYx8U; SMSESSION=OYXcovF1l577odr3Y+kx2fNPHEz4GXkgalxer6G3swSYF9BPTNTZQUlqK7eFhGYB5MvDcACBBx5KrE95MCbcDAuLDmyDd9eZ/p65ZuvmFN5U8axzkwIMI4tDS1Ga/LEV6NOauQldQtvWpYPEKF6yynMbCNVqg1xaKnog711LPt4L7/aOiCM1Hq598SpRf5h0m0zWc6Jji0vUvXWmstqQ2jJw7ISwMTW6Nqztyv/F/DmW9Th0blejuqJeWxrkR0JXgnI6YmvSpdd/HpGrGeY0v2jv2DzM+v+R4D1ZO0W2DGeiDdY5IxGnmTToG7KTSRf4zZSRZvLWFZPkSK8hWx9j2D7uKe50Ws2IYe+TkakO4d0tKorc3dajkmi7M3PA1rl5EiMUWjwKfRBVybwJhcSh/XNEfMTyV7VChDd5n6iTSNYRZOwb63HAZ47guhoFforrWG4yiDEPmRZeCW1MKzspRKHQWH8RIoyY1/+lrR59xlcTb/awXGPxmkE+zqQKXmZciImMZSm/F0iQXYbScJB5NUaMoQ5aMMIy8uTrKa+Q09bpj3xyjKHQ+GPqTULXfL7rZjrg7mm2MnOFVeD9OFBZPhvbnvJ0sK1CElzkSc+/o9r0z+rPpiOwAn+F40KucjP1ynDvWxO63cCa1ge5iZyLocDgekMSB+h62zj2Yjp03qKs199V9rBvZV91bmBFk8dlOLkxz1uZXqca/7jneKXuHm9j56n8lZK2tE/26Xwn+jwDRepBzG73/GsPg5vkptFlctrzTzI0zFJfcLOyR5qJR5qlJZfU+REkV+tbH7yQjcVk76xrtYNV5J6ixLh8gEIozBy2EJQuBW6Sg+7tX0uBsHxgU7Kq0l6OsUAkqq07jJ7nv4da7JNqK5jxr1KnKK+X9Cgnjzfm/nBvR7Tv2/kmyZLd/A8pZ356+U+sMiRhFIYEvp3ayyIBeiBnCENvo0bpyHcEwelVj7RiEyIFVzaWs0ShuhMBZjxTCPQcUDrVG/MKtcozwzHg0CSldtK7TRqbEcIzGWxYvLl35Nwn8DblPNOk/LYbpSmG3ivqLMyHCHzTM39r4X0e2nOe/tHRJq/6JWxLWcC8fQ0T1llbqIqurkJ9iV3Oozpn5zELtArbd/SiHBzjJzSRPlxJcVAmwp4XnXchrNw05on/R4cd7xSPRXN2L4n4h7Wv9DaF430uPjoX7p0Eow8ykeqJ7HXQAnik4R29HKrsvD0eb4Hs2tUxSQBuYJlp0UrFb8IrXFCiOnOJRrfvVeCJjWm9A6+Y6+m2zdXVDNe8X6vz4VqTSNxWTRquFxOf5MLcwsOAh4ijaraV6jT1yY9TdT05H1ekfdOu; 52867=US; s_dfa=ihgusglobal%2Cihgsuper; X-IHG-SRV=iadd1plb2cwb001; AMCV_8EAD67C25245B1870A490D4C%40AdobeOrg=1099438348%7CMCMID%7C34852208300861905892616333797976260600%7CMCAAMLH-1539122626%7C7%7CMCAAMB-1539204973%7CRKhpRz8krg2tLO6pguXWp5olkAcUniQYPHaMWWgdJ3xzPWQmdj0y%7CMCOPTOUT-1538607373s%7CNONE%7CMCAID%7C2D94112B851D5158-600001628000F567%7CMCSYNCSOP%7C411-17792%7CvVersion%7C2.1.0%7CMCIDTS%7C17808; QSI_HistorySession=https%3A%2F%2Fwww.ihg.com%2Fhotels%2Fus%2Fen%2Ffind-hotels%2Fhotel%2Frates%3FqDest%3DAtlanta%2C%2520GA%2C%2520United%2520States%26qCiMy%3D92018%26qCiD%3D14%26qCoMy%3D92018%26qCoD%3D18%26qAdlt%3D1%26qChld%3D0%26qRms%3D1%26qRtP%3DIVEDI.6CBARC.IMTDY.IVANI%26qCpid%3D954408789%26qIta%3D99504425%26qSlH%3DATLPW%26qSlRc%3DTQNN%26qAkamaiCC%3DUS%26qSrt%3DsDD%26qBrs%3Dic.in.vn.cp.hi.ex.rs.cv.sb.cw.ma.ul.ki.va%26qAAR%3DIVEDI.6CBARC.IMTDY%26qWch%3D0%26qSmP%3D1%26qRad%3D30~1538493921669%7Chttps%3A%2F%2Fwww.ihg.com%2Fhotels%2Fus%2Fen%2Fpay%2Fpayment~1538493932956%7Chttps%3A%2F%2Fwww.ihg.com%2Fhotels%2Ffr%2Ffr%2Fpay%2Fpayment~1538493956926%7Chttps%3A%2F%2Fwww.ihg.com%2Fhotels%2Fus%2Fen%2Ffind-hotels%2Fhotel%2Flist%3FqDest%3DAtlanta%2C%2520GA%2C%2520United%2520States%26qCiMy%3D92018%26qCiD%3D2%26qCoMy%3D92018%26qCoD%3D3%26qAdlt%3D1%26qChld%3D0%26qRms%3D1%26qRtP%3DIVEDI.6CBARC%26qCpid%3D954408789%26qIta%3D99504425%26qAkamaiCC%3DUS%26qSrt%3DsDD%26qBrs%3Dic.in.vn.cp.hi.ex.rs.cv.sb.cw.ma.ul.ki.va%26qAAR%3DIVEDI.6CBARC%26srb_u%3D1%26qRad%3D30~1538494114986%7Chttps%3A%2F%2Fwww.ihg.com%2Fhotels%2Fus%2Fes%2Ffind-hotels%2Fhotel%2Flist%3FqDest%3DAtlanta%2C%2520GA%2C%2520United%2520States%26qCiMy%3D92018%26qCiD%3D2%26qCoMy%3D92018%26qCoD%3D3%26qAdlt%3D1%26qChld%3D0%26qRms%3D1%26qRtP%3DIVEDI.6CBARC%26qCpid%3D954408789%26qIta%3D99504425%26qAkamaiCC%3DUS%26qSrt%3DsDD%26qBrs%3Dic.in.vn.cp.hi.ex.rs.cv.sb.cw.ma.ul.ki.va%26qAAR%3DIVEDI.6CBARC%26srb_u%3D1%26qRad%3D30~1538575372862%7Chttps%3A%2F%2Fwww.ihg.com%2Fhotels%2Fus%2Fen%2Ffind-hotels%2Fhotel%2Flist%3FqDest%3DAtlanta%2C%2520GA%2C%2520United%2520States%26qCiMy%3D92018%26qCiD%3D2%26qCoMy%3D92018%26qCoD%3D3%26qAdlt%3D1%26qChld%3D0%26qRms%3D1%26qRtP%3DIVEDI.6CBARC%26qCpid%3D954408789%26qIta%3D99504425%26qAkamaiCC%3DUS%26qSrt%3DsDD%26qBrs%3Dic.in.vn.cp.hi.ex.rs.cv.sb.cw.ma.ul.ki.va%26qAAR%3DIVEDI.6CBARC%26srb_u%3D1%26qRad%3D30~1538600174841; uhf_userstate=IMPLICIT; prevPageName=6C%3A%20SEARCH%20RESULTS; mbox=PC#3f8a1d1545ee42f3839767e1ceec90dd.17_29#1546282444|session#717396900d264a78853597617adfaf65#1538602175; __CT_Data=gpv=2633&ckp=tld&dm=ihg.com&apv_16508_www03=3597&cpv_16508_www03=3597&rpv_16508_www03=3537; X-IHG-GAPI-SRV=iadd1plapiwb007; ADRUM=s=1538600354237&r=https%3A%2F%2Fwww.ihg.com%2Fhotels%2Fus%2Fen%2Ffind-hotels%2Fhotel%2Flist%3F1384005878' --compressed

**Comment by abhijitkane** [comment]
@ingabraham I'm unable to reproduce this - I was able to import successfully on Postman 6.4.1 for Mac. Can you attach a screenshot of the console errors (View > Developer Tools > Current View > Console) here?

<img width="1005" alt="screen shot 2018-10-05 at 11 46 47" src="https://user-images.githubusercontent.com/681190/46518987-629d6f80-c894-11e8-9c41-aad52fe4cfd8.png">

**Comment by ingabraham** [comment]
> @ingabraham I'm unable to reproduce this - I was able to import successfully on Postman 6.4.1 for Mac. Can you attach a screenshot of the console errors (View > Developer Tools > Current View > Console) here?
> 
> <img alt="screen shot 2018-10-05 at 11 46 47" width="1005" src="https://user-images.githubusercontent.com/681190/46518987-629d6f80-c894-11e8-9c41-aad52fe4cfd8.png">

@abhijitkane I am not longer to replicate this issue this morning using the same cURL I shared,
it is working for me as well, it seems is fixed now

**Comment by pernielsentikaer** [comment]
I have the same issue - trying to import this:
`curl 'https://test.dk' -H 'Origin: https://test.dk' -H 'Accept-Encoding: gzip, deflate, br' -H 'Accept-Language: en-US,en;q=0.9,da-DK;q=0.8,da;q=0.7,mt;q=0.6' -H $'Cookie: CookieTestConsent={stamp:\'tzfBqDpUsH4NEDbuq3dfjNxmrefdsfdsFSDFb3Q9ueTrxDHcoRErUYgsonbCRNQcLA==\'%2Cnecessary:true%2Cpreferences:true%2Cstatistics:true%2Cmarketing:true%2Cver:1}; _ga=GA1.4.10fds5rew8608474.153976rewfds5958; _gid=GA1.4.592fds5528rew08.15402908rewfds37; fdsfdsf=konfirewgurator%2Ftype_id%2F%2Fdessin%2F%2FmountHeights%2Fnull%2Fmarking%2F; form_key=J7CwaYXhdRW78hQL; mage-cache-storage=%7B%7D; mage-cache-storage-section-invalidation=%7B%7D; mage-cache-sessid=true; recently_viewed_product=%7B%7D; recently_viewed_product_previous=%7B%7D; recently_compared_product=%7B%7D; recently_compared_product_previous=%7B%7D; product_data_storage=%7B%7D; form_key=J7CwaYXhdRW78hQL; rew=n1jxEUCEMerewvMEgC5EyAH6bWEKT197L7L; X-Magento-Vary=b06b00ff22275b9rewb1erew0e5b657e72c0be085f45e1; section_data_ids=%7B%22messages%22%3A8000%7D; PHPSESSID=u4npg1h3mrew9b4a9eoib385ival4; store=rewrewr; connect.sid=s%3ALAASIuv4mrewVvwKmgJGpjgyaEpYSkUUMYk.%2FJel15xdpUzzr6kLbwIi4NbrewJVJtWj1ZP%2B5WuKOkUN86c; private_content_version=35288977a7rew47re0151c94d49f4d7e30b83' -H 'Connection: keep-alive' -H 'Pragma: no-cache' -H 'pnt: 1' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.67 Safari/537.36' -H 'Content-Type: application/json' -H 'Accept: application/json, text/plain, */*' -H 'Cache-Control: no-cache' -H 'Referer: https://test.dk' -H 'DNT: 1' -H 'x-api-key: mujyyLFbXs70ROHG36GhOw3O7Rq8woe1g6muVqV8' --data-binary '{"key":"TEST","websiteId":2,"storeId":4,"remove":true,"coupon":{"code":"erwrwer"}}' --compressed`

**Comment by madskunker** [comment]
I have the same issue
 6.7.4 Mac OSX
`curl 'https://developer.intuit.com/v2/oauth2playground/Oauth2AccessTokenFromRefreshToken?timestamp=1551870118095' -H $'cookie: ivid_synced=true; tracking_user_id=193514860459379; qbo.clientType=web; split_test_version=test100; s_cc=true; s_fid=440666371FF736A5-199B11D17ED9B39C; s_vi=[CS]v1|2E3F80BE8507D88B-6000010BC0000456[CE]; ac_s_v8.5_appcenter=bdva4kabl0wtqmnmirjvaozu; UserLocale=en-US; optimizelyEndUserId=oeu1551827333286r0.17797178070011088; ius_session=D8ED069AD4514E3391A29FD93C5D1C38; did=SHOPPER2_718be2e84d4bc81d83f685caf79ef885c12480442f1a7937f5f715cebeab58de9db91e7674d4097d61cd96af59e150ac; ivid_b=cd4fadf1-948c-492a-b6ce-dc9cc6aecdf1; ivid=7f327295-f48a-4aad-8664-bdb39499eb17; qbn.ticket=V1-85-b0qate6w2mh6mnn25biff9; qbn.tkt=V1-85-b0qate6w2mh6mnn25biff9; qbn.authid=193514860459379; qbn.gauthid=193514860459379; qbn.agentid=193514860459379; qbn.parentid=50000003; qbn.glogin=jeff%40swxl.io; userIdentifier=193514860459379; isDevXUser=true; ADRUM=s=1551827685500&r=https%3A%2F%2Faccounts.intuit.com%2Fsignup-confirmation.html%3F2117793462; propertySegments=1551828234032%7cAC%3a1%3a%3a; __utma=1.248059611.1551828237.1551828237.1551828237.1; __utmc=1; __utmz=1.1551828237.1.1.utmcsr=developer.intuit.com|utmccn=(referral)|utmcmd=referral|utmcct=/v2/ui; ursvar=developer.intuit.com%20%5Bref%5D; ajs_user_id=null; ajs_group_id=null; ajs_anonymous_id=%227e7748a4-4cb2-4f85-8a78-d9613e20c7d7%22; amplitude_idundefinedintuit.com=eyJvcHRPdXQiOmZhbHNlLCJzZXNzaW9uSWQiOm51bGwsImxhc3RFdmVudFRpbWUiOm51bGwsImV2ZW50SWQiOjAsImlkZW50aWZ5SWQiOjAsInNlcXVlbmNlTnVtYmVyIjowfQ==; amplitude_id_1e35cf9786780b57cf805cd80925b103intuit.com=eyJkZXZpY2VJZCI6Ijg3ODBjNzRmLTUwNDctNGEzYS1hMjA0LWM2ZTM4OWI0ZGVlYVIiLCJ1c2VySWQiOm51bGwsIm9wdE91dCI6ZmFsc2UsInNlc3Npb25JZCI6MTU1MTgyODIzNzg2MiwibGFzdEV2ZW50VGltZSI6MTU1MTgyOTk2NzYxMiwiZXZlbnRJZCI6MCwiaWRlbnRpZnlJZCI6Mywic2VxdWVuY2VOdW1iZXIiOjN9; s_cpm=%5B%5B\'developer.intuit.com%20%5Bref%5D\'%2C\'1551829969584\'%5D%5D; AppCenter_Ticket=6_bpfjqnyw7_chkpzh_b_dqknzrh3wueh5d3yy2hr73fsqz_78amd49y2rkp3ctbbbekx3vpg6z2g8suxpvmnfr7sedrpr4ss36b_bpfifgme7; __RequestVerificationToken=xfk9tCyWI7H4-6-sXx1oUcb1D5R_rTpdX7lr2ePt-Jz_lhskd5sr5Bve_WjXjMnOG34aL62V6RPpa6UbvrShJnE5Pdo1; XSRF-TOKEN=hl5mBo0wI0Kj06qo5eYPDOKli9WokQLhjWcVkEh0d0oWQF7Wh0PRtynzwKme9U654B5cNOWPlbBVSQNnxDC2_2osixUoP7xNq9S8HHarhPur0ngy0; qboeuid=39f82919.58361cfd67117; s_sq=intuitdeveloper%2Cintuitctoiux%3D%2526pid%253Dhttps%25253A%25252F%25252Fdeveloper.intuit.com%25252Fv2%25252Fui%252523%25252Fplayground%25253Fstate%25253DPlaygroundAuth%252526code%25253DL0115518303299jsUIHCbKCpwAuutg83pk0bj5YtjdqiaL9JkB%252526realmId%25253D123146330799669%2526oid%253DRefresh%252520Access%252520Token%2526oidt%253D3%2526ot%253DSUBMIT' -H 'origin: https://developer.intuit.com' -H 'x-xsrf-token: hl5mBo0wI0Kj06qo5eYPDOKli9WokQLhjWcVkEh0d0oWQF7Wh0PRtynzwKme9U654B5cNOWPlbBVSQNnxDC2_2osixUoP7xNq9S8HHarhPur0ngy0' -H 'accept-language: en-US,en;q=0.9' -H 'user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36' -H 'content-type: application/json;charset=UTF-8' -H 'accept: application/json, text/plain, */*' -H 'referer: https://developer.intuit.com/v2/ui' -H 'authority: developer.intuit.com' -H 'accept-encoding: gzip, deflate, br' --data-binary '{"refreshToken":"L011560556397MaOulTCxuWPjRkqeZNPlbAU9YKJ34mNod9mHd","cliId":"L04xWZydljFKqGzmbHBeYPfWQcNayz4kM7lojhzHO93Aoa55yl","cliSecret":"YvNVy4MaXqYa2Ktu4KlT7ffZjRSqbz3H7QPMNfmJ"}' --compressed`

**Comment by jramirezpr** [comment]
Same issue

**Comment by faisalmirza** [comment]
Same issue. Anyone found a workaround?

**Comment by jkshobby** [comment]
Issue is due to header is bigger than postman can handle.  Manually copy paste the headers, urls etc. this should solve.

**Comment by abhijitkane** [comment]
@madskunker @pernielsentikaer I'm able to import both cURL snippets in the latest Postman version - 7.2.2. Closing this for now.

**Comment by n2nco** [comment]
Same issue.

**Comment by parsalotfy** [comment]
same issue

**Comment by VivekDabhi** [comment]
Same issue

Error while importing Curl: arg.startsWith is not a function

**Comment by rinat-enikeev** [comment]
same issue

**Comment by GOVYANSONG** [comment]
FYI I had the same issue initially. this is what worked for me: in chrome, right click the ajax call and select copy as curl (bash), then paste as raw text in postman.

**Comment by ConnorDillonDev** [comment]
same issue

**Comment by lbertoncin** [comment]
same issue for me

**Comment by hb20007** [comment]
I was having the same issue and I realized that Chrome was giving a wrong cURL command when using the Copy as cURL feature in the Network tab. I tried to copy the same request as cURL using Firefox and I was able to successfully import it into Postman.

**Edit**: I realized that I had accidentally clicked "copy all as cURL" in Chrome, instead of "copy as cURL". If I use "copy as cURL" in Chrome, it works fine.
